frontend-react/src/pages/admin/AdminRevHistory.tsx

```diff
@@ -0,0 +1,273 @@
+import { Helmet } from "react-helmet";
+import React, { Suspense, useCallback, useState } from "react";
+import { Grid, GridContainer, Accordion } from "@trussworks/react-uswds";
+import { AccordionItemProps } from "@trussworks/react-uswds/lib/components/Accordion/Accordion";
+import { useParams } from "react-router-dom";
+
+import HipaaNotice from "../../components/HipaaNotice";
+import {
+    SettingRevision,
+    SettingRevisionParams,
+    useSettingRevisionEndpointsQuery,
+} from "../../network/api/Organizations/SettingRevisions";
+import Spinner from "../../components/Spinner";
+import { AuthElement } from "../../components/AuthElement";
+import { MemberType } from "../../hooks/UseOktaMemberships";
+import { jsonSortReplacer } from "../../utils/JsonSortReplacer";
+import { formatDate } from "../../utils/misc";
+import { StaticCompare } from "../../components/StaticCompare";
+
+/** Accordion components need data in specific format **/
+const dataToAccordionItems = (props: {
+    key: string; // used for React key and passed back to the onClickHandler
+    selectedKey: string;
+    onClickHandler: (
+        key: string,
+        itemClickedKey: string,
+        data: SettingRevision
+    ) => void;
+    data: SettingRevision[];
+}): AccordionItemProps[] => {
+    const results: AccordionItemProps[] = [];
+    if (props.data.length === 0) {
+        return [];
+    }
+
+    // should come back sorted by name and version from server. Sort by name, then by version
```

Comment:
if they come back sorted why are we sorting again?

Reply:
I tend to never trust server API results blindly (even if I control the client and server). Burned too many times with something seemingly sorted correctly, but then someone else comes along and removes the sort to speed up the query (or it happened to be sorted correctly but it's a natural sort and someone restores the DB and the order gets weird). 

It's not a lot of items, so I figured why not.

Reply:
sound reasoning. the comment just threw me off